react-app/manifest.ts

```diff
@@ -18,7 +18,7 @@ const manifest: chrome.runtime.ManifestV3 = {
   action: {
     default_popup: 'src/pages/popup/index.html',
     default_title: 'YT Plugin',
-    default_icon: 'icon-34.png',
+    default_icon: 'icon-32.png',
```

Comment:
```suggestion
    default_icon: 'logo-32.png',
```

And do it for other names